Resources/Prototypes/_SpaceCats/Entities/Structures/Machines/survelliance_camera_routers.yml

```
@@ -0,0 +1,9 @@
+- type: entity
+  parent: SurveillanceCameraRouterBase
+  id: SurveillanceCameraRouterHandheld
+  name: Camerahelds
+  description:
+  components:
+    - type: SurveillanceCameraRouter
+      subnetFrequency: SurveillanceCameraHandheld
+      subnetColor: "#DE20AA"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)